# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../chrony/chrony_conf.patch
# Copyright (C) 2004 - 2010 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- chrony-1.24/conf.c.vanilla	2010-02-10 10:10:36.000000000 +0100
+++ chrony-1.24/conf.c	2010-02-10 10:11:04.000000000 +0100
@@ -59,7 +59,7 @@
 /* ================================================== */
 
 #ifndef DEFAULT_CONF_DIR
-#define DEFAULT_CONF_DIR "/etc"
+#define DEFAULT_CONF_DIR "/etc/chrony"
 #endif
 
 #define DEFAULT_CONF_FILE DEFAULT_CONF_DIR"/chrony.conf"
